internal/provisioner/plain/controllers/bundleinstance_controller.go

```diff
@@ -51,6 +51,17 @@ import (
 	"github.com/operator-framework/rukpak/internal/util"
 )
 
+const (
+	maxGeneratedBundleLimit = 4
```

Comment:
Should this be configurable with a default? How did we settle on 4?

Reply:
This is just a short term bandaid to prevent hotlooping behavior before we start implementing revision history limit behavior, which the deployment/replicaset controllers use.